tasks/node.yml

```diff
@@ -2,12 +2,8 @@
 
 - name: SSH keys
   authorized_key:
-    key: "{{ hostvars['OPENSHIFT_ANSIBLE_HOST']['public_key'] }}"
+    key: "{{ hostvars[item]['public_key'] }}"
     user: root
-    key_options: 'from="{{ openshift_ssh_from }}"'
-  when: hostvars['OPENSHIFT_ANSIBLE_HOST']['public_key'] is defined
-
-- name: SSH key warning
-  debug:
-    msg: SSH key not added to node because the Ansible host wasn't included in the run
-  when: hostvars['OPENSHIFT_ANSIBLE_HOST']['public_key'] is not defined
+    key_options: from="{{ hostvars[item]['ansible_default_ipv4']['address'] }}"
+  with_items: "{{ groups['openshift_ansible_hosts'] }}"
```

Comment:
`with_items` is deprecated in favor of `loop`

Reply:
Done